functional/FunctionalTest.java

```diff
@@ -1994,11 +2004,22 @@ public static void testGetBucketPolicy(String objectPrefix, PolicyType policyTyp
       if (type != policyType) {
         throw new Exception("[FAILED] Expected: " + policyType + ", Got: " + type);
       }
-      mintSuccessLog("getBucketPolicy(String bucketName, String objectPrefix)", null, startTime);
+      mintSuccessLog("" + policyType + ": getBucketPolicy(String bucketName, String objectPrefix)", null, startTime);
```

Comment:
In mint, `function` is logged as a string with the name of function being tested (getBucketPolicy) in this case. Adding `policyType` is not helpful.

Reply:
Could you tell me why it is not useful by seeing four repeated logs with
```
getBucketPolicy(String bucketName, String objectPrefix)
getBucketPolicy(String bucketName, String objectPrefix)
getBucketPolicy(String bucketName, String objectPrefix)
getBucketPolicy(String bucketName, String objectPrefix)
``` ?

Reply:
Yes, as we discussed earlier in the call

- json logs in Mint are for parsing by other tools.
- As the `function` entry in the log is meant to hold the API being tested, it is easy to parse and find out stuff like - which API has how many tests, which API failed, etc. Adding custom prefix will make parsing difficult.

Additionally, I don't think this is helpful as
- `PASS` logs are generally not going to be looked at.
- And if something fails, there is the error field with detailed stack trace with line number and other details.

Reply:
As per this discussion in slack, I defer this comment

Reply:
I am relaxing my decision and make things done